library/private/options_tools.h

```diff
@@ -206,7 +206,7 @@ color_t parse(const std::string& str)
 //----------------------------------------------------------------------------
 /**
  * Parse provided string into a direction_t.
- * Supported formats: "X,Y,Z", "[+|-][X|Y|Z]"
+ * Supported formats: "X,Y,Z", "[[+|-]X][[+|-]Y][[+|-]Z]"
```

Comment:
please precise that order should be respected and case does not matter ?

Reply:
order is already be implied by this syntax, no?

Reply:
true.